pom.xml

```diff
@@ -16,7 +16,7 @@
 
   <properties>
     <hadoop.version>2.6.0</hadoop.version>
```

Comment:
Consider updating the Hadoop version to match that declared by Hive 2.3.4. See https://github.com/apache/hive/blob/rel/release-2.3.4/pom.xml#L141

Reply:
We tried but ran into all kinds of issues with the version of Hadoop that Cascading and Tez support which meant we couldn't move to 2.7.2.